### PR TITLE
Update support

### PR DESCRIPTION
**Pull request by yukiexe**
感覺是我桌面 Scatter 出了問題，單純用裡面的轉帳也會卡住
我測這個也是確認視窗跳完就石沉大海了
已經不知道到底是哪的問題的

給你們測吧

### CODE REVIEW

**Comment by yukiexe** [comment]
送不出去，怪
圖片

**Comment by lychees** [comment]
Will do.

**Comment by lychees** [comment]
The bug of support() function has been resolved.
It works for now.

Can I merge withdraw() and action_claim() function as well?

**Comment by yukiexe** [comment]
not work still :(

File: src/views/Article.vue [hunk 1]
```diff
@@ -78,7 +78,20 @@ export default {
...
     },
     updateTitle() {
-
+    },
+    compiledMarkdown() {
+      return markdownIt.render(this.post.content);
+    },
+    getClipboard() {
+      const {currentUsername, scatterAccount} = this;
+      // todo(minakokojima): figure out what is the different between following variables.
```
Comment (frankwei98) [inline src/views/Article.vue]: no difference here
`currentUsername` is the projection of `scatterAccount.name` which you can find out [here](https://github.com/smart-signature/smart-signature-future/blob/4b618f04928eee504e26614d9c4a8135ea5e8afc/src/store.js#L21)
```js
currentUsername: ({ scatterAccount }) => (scatterAccount ? scatterAccount.name : null),
```

**Pull request merged**